pkg/analysis/value.go

```diff
@@ -376,7 +376,12 @@ type Resolver interface {
 	Import(from, path string) ast.Node
 }
 
-func NodeToValue(node ast.Node, resolver Resolver) (res *Value) {
+var maxStackDepth = 100
```

Comment:
could we bump this a bit higher to something like 300? There's a lot of legit high-depth logic (because each ast element counts for a stack entry)